ReactApp/src/utils/rest-api-call.js

```diff
@@ -38,6 +38,8 @@ export const serverPath = {
     EDIT_META_QUESTION: 'editMetaQuestion',
     GET_META_QUESTIONS_FOR_APPENDIX: 'getMetaQuestionForAppendix',
     REFRESH_TOKEN: 'refreshJWT',
+    GENERATE_TASK: 'generateTask',
+    COMPLETE_GENERATED_TASK: 'completeGeneratedTask',
```

Comment:
Please add to pathToReturnTypeMap the expected return type

Reply:
I didn't understand, can you elaborate? @OfekN11

Reply:
Like the validateParameters there is same thing just for the API response type.
requestpathToReturnTypeMap  is in the  api...js file, and it is the map from the api request to the expected return type from the server